node/api.go

```diff
@@ -342,3 +343,57 @@ func (s *publicWeb3API) ClientVersion() string {
 func (s *publicWeb3API) Sha3(input hexutil.Bytes) hexutil.Bytes {
 	return crypto.Keccak256(input)
 }
+
+type ExecutionPoolThreads struct {
+	HttpLimit uint64
+	WSLimit   uint64
+}
+
+type ExecutionPoolRequestTimeout struct {
+	HttpLimit time.Duration
+	WSLimit   time.Duration
+}
+
+func (api *privateAdminAPI) GetExecutionPoolThreads() *ExecutionPoolThreads {
+	httpLimit := api.node.http.httpConfig.threads
+	wsLimit := api.node.ws.wsConfig.threads
+
+	executionPoolThreads := &ExecutionPoolThreads{
+		HttpLimit: httpLimit,
+		WSLimit:   wsLimit,
+	}
+
+	return executionPoolThreads
+}
+
+func (api *privateAdminAPI) GetExecutionPoolRequestTimeout() *ExecutionPoolRequestTimeout {
+	httpLimit := api.node.http.httpConfig.requesttimeout
```

Comment:
will `ChangeSize, ChangeTimeout` change this config values as well? If not we need either to start changing them or read actual values from SafePool.